omod/src/main/webapp/fragments/systemadmin/accounts/personDetails.gsp

```diff
@@ -23,7 +23,7 @@
     <div class="adminui-form-person-details-panel">
 <% } %>
         <fieldset class="<% if(createAccount) { %>adminui-account-fieldset<% } else { %>adminui-form-person-details-panel<% } %>">
-        <legend><b>${ ui.message("adminui.person.details") }</b></legend>
+        <legend class="w-auto"><b>${ ui.message("adminui.person.details") }</b></legend>
```

Comment:
what does it mean by `w-auto`? is it `width-auto`?

Reply:
Hi @suthagar23 yes these are predefined tags now in bootstrap.

For more ref.: -
https://getbootstrap.com/docs/4.1/utilities/sizing/

Reply:
@suthagar23 Is this good to go?